src/components/AccountDetails/index.tsx

```diff
@@ -265,11 +265,13 @@ export default function AccountDetails({
               <AccountGroupingRow>
                 {formatConnectorName()}
                 <div>
-                  {connector !== injected && connector !== walletlink && (
+                  {/* coinbase wallet reloads the page right now, which breaks the walletOverride from being set properly on localStorage */}
```

Comment:
![image](https://user-images.githubusercontent.com/4899429/167901455-9f1ce76a-3872-4f3c-9f56-4b785b83e4d6.png)
some more context here. felt it'd be best to wait for them to support this instead of adding specific logic now. i will fix comment